apps/openssl.cnf

```diff
@@ -11,9 +13,31 @@
 # defined.
 HOME			= .
 
+ # This must be in the default section
+openssl_conf = openssl_init
+
+# For FIPS
+# Optionally include a file that is generated by fipsinstall that contains
+# configuration data required by the OpenSSL fips provider.
+# Refer to the OpenSSL security policy for more information.
+# .include fipsmodule.cnf
+
+[openssl_init]
+providers = provider_sect
+
+# List of providers to load
+[provider_sect]
+default = default_sect
+# The fips section name should match the section name inside the
+# included fipsmodule.cnf.
+# fips = fips_sect
+
+[default_sect]
+# activate = 1
+oid_section = new_oids
```

Comment:
Wait, what?  This line should remain in the default section.

Reply:
For some reason that made some of the tests actually hang..

Reply:
I will debug it to see what is going on..

Reply:
Not sure why but today it is working.. All I did was a make update and a rebuild.. weird.